From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 17 Aug 2026 07:00:43 +0000
Subject: Translated using Weblate (Turkish)

Currently translated at 0.0% (0 of 3 strings)

Translation: openkylin-huanghe/peony-bluetooth-plugin
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/peony-bluetooth-plugin/tr/
---
 peony-bluetooth-plugin/translations/peony-bluetooth-plugin_tr.ts | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/peony-bluetooth-plugin/translations/peony-bluetooth-plugin_tr.ts b/peony-bluetooth-plugin/translations/peony-bluetooth-plugin_tr.ts
index 1a2895a..f13664c 100644
--- a/peony-bluetooth-plugin/translations/peony-bluetooth-plugin_tr.ts
+++ b/peony-bluetooth-plugin/translations/peony-bluetooth-plugin_tr.ts
@@ -6,17 +6,17 @@
     <message>
         <location filename="../bluetoothplugin.cpp" line="165"/>
         <source>Send from bluetooth to...</source>
-        <translation type="unfinished"></translation>
+        <translation>Bluetooth ile dosya gönder...</translation>
     </message>
     <message>
         <location filename="../bluetoothplugin.h" line="54"/>
         <source>Peony-Qt bluetooth Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>Bluetooth eklentisi</translation>
     </message>
     <message>
         <location filename="../bluetoothplugin.h" line="55"/>
         <source>Bluetooth Menu Extension</source>
-        <translation type="unfinished"></translation>
+        <translation>Bluetooth menü uzantısı</translation>
     </message>
     <message>
         <source>bluetooth Menu Extension</source>
